scooby/report.py

```diff
@@ -201,13 +201,13 @@ def __repr__(self):
         text += date_text+'\n'
 
         # ########## Platform/OS details ############
-        text += '{:>18}'.format(self.system)+' : OS\n'
-        text += '{:>18}'.format(self.cpu_count)+' : CPU(s)\n'
-        text += '{:>18}'.format(self.machine)+' : Machine\n'
-        text += '{:>18}'.format(self.architecture)+' : Architecture\n'
+        text += '{:>21}{}'.format('OS : ', self.system)+'\n'
```

Comment:
Would you mind changing all (lines 204-207, 209, 210)
- `{:>21}{}` to `{:>18} : {}\n`
and then remove later in the line the
-  ` : `, and 
- `+'\n'`?
 This way it would all be consistent and nice. It would even be more consistent than before!

Thanks for your contribution!

Reply:
(Basically change it to the format as used in `Loop over packages`.)